command/acl/idp/idp.go

```diff
@@ -0,0 +1,58 @@
+package idp
+
+import (
+	"github.com/hashicorp/consul/command/flags"
+	"github.com/mitchellh/cli"
+)
+
+func New() *cmd {
+	return &cmd{}
+}
+
+type cmd struct{}
+
+func (c *cmd) Run(args []string) int {
+	return cli.RunResultHelp
+}
+
+func (c *cmd) Synopsis() string {
+	return synopsis
+}
+
+func (c *cmd) Help() string {
+	return flags.Usage(help, nil)
+}
+
+const synopsis = "Manage Consul's ACL Identity Providers"
+const help = `
+Usage: consul acl idp <subcommand> [options] [args]
+
+  This command has subcommands for managing Consul's ACL Identity Providers.
+  Here are some simple examples, and more detailed examples are available
+  in the subcommands or the documentation.
+
+  Create a new ACL Identity Provider:
+
+      $ consul acl idp create -type kubernetes \
+                              -name "kube-1" \
+                              -description "This is an example kube idp"
```

Comment:
Is this a useful example? Is that sufficient to create the provider? How does Consul know where the api server is or what credentials? Do those default from ENV or something? I assume yes to all but might be worth noting in the example commentary that that is what this is doing?